app-misc/lcdproc/lcdproc-0.5.9.ebuild

```diff
@@ -0,0 +1,206 @@
+	freetype? ( lcd_devices_glcd )
+	png? ( lcd_devices_glcd )"
+
+# Define the list of valid lcd devices.
+# Some drivers were removed from this list:
+# - svga: It needs media-libs/svgalib, which is masked and obsolete.
+IUSE_LCD_DEVICES=( bayrad CFontz CFontzPacket curses CwLnx ea65
+	EyeboxOne futaba g15 glcd glcdlib glk hd44780 i2500vfd
+	icp_a106 imon imonlcd IOWarrior irman irtrans
+	joy lb216 lcdm001 lcterm linux_input lirc lis MD8800 mdm166a
+	ms6931 mtc_s16209x MtxOrb mx5000 NoritakeVFD
+	Olimex_MOD_LCD1x9 picolcd pyramid rawserial
+	sdeclcd sed1330 sed1520 serialPOS serialVFD
+	shuttleVFD sli stv5730 SureElec t6963 text
+	tyan ula200 vlsys_m428 xosd yard2LCD )
+
+# Add supported drivers from 'IUSE_LCD_DEVICES' to 'IUSE' and 'REQUIRED_USE'
+REQUIRED_USE+=" || ( "
+for LCD_DEVICE in "${IUSE_LCD_DEVICES[@]}"; do
+	LCD_DEVICE="${LCD_DEVICE,,}"
+	IUSE+=" lcd_devices_${LCD_DEVICE} "
+	REQUIRED_USE+=" lcd_devices_${LCD_DEVICE} "
+done
+REQUIRED_USE+=" ) "
+unset LCD_DEVICE
+
+# Define dependencies for all drivers in 'IUSE_LCD_DEVICES'
+DEPEND_LCD_DEVICES="lcd_devices_cfontz? ( dev-libs/libhid:= )
+	lcd_devices_cfontzpacket? ( dev-libs/libhid:= )
+	lcd_devices_cwlnx? ( dev-libs/libhid:= )
+	lcd_devices_futaba? ( virtual/libusb:1= )
+	lcd_devices_g15? ( app-misc/g15daemon
+		dev-libs/libg15render:=
+		virtual/libusb:0= )
+	lcd_devices_glcd? ( app-misc/glcdprocdriver:=
+		dev-embedded/libftdi:1=
+		dev-libs/libhid:=
+		dev-libs/serdisplib:=
+		virtual/libusb:0=
+		x11-libs/libX11:= )
+	lcd_devices_hd44780? ( dev-embedded/libftdi:1=
+		dev-libs/libugpio:=
+		virtual/libusb:0= )
+	lcd_devices_i2500vfd? ( dev-embedded/libftdi:1= )
+	lcd_devices_irman? ( media-libs/libirman:= )
+	lcd_devices_iowarrior? ( virtual/libusb:0= )
+	lcd_devices_lirc? ( app-misc/lirc )
+	lcd_devices_lis? ( dev-embedded/libftdi:1= )
+	lcd_devices_lb216? ( dev-libs/libhid:= )
+	lcd_devices_mdm166a? ( dev-libs/libhid:= )
+	lcd_devices_mtc_s16209x? ( dev-libs/libhid:= )
+	lcd_devices_mx5000? ( app-misc/mx5000tools )
+	lcd_devices_noritakevfd? ( dev-libs/libhid:= )
+	lcd_devices_picolcd? ( virtual/libusb:1= )
+	lcd_devices_shuttlevfd? ( virtual/libusb:0= )
+	lcd_devices_ula200? ( dev-embedded/libftdi:1= )
+	lcd_devices_xosd? ( x11-libs/libX11:=
+		x11-libs/xosd:= )"
+
+RDEPEND="${DEPEND_LCD_DEVICES}"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[dot]
+		app-text/xmlto )
+	freetype? ( media-libs/freetype:2= )
+	nfs? ( net-fs/nfs-utils )
+	png? ( media-libs/libpng:0= )
+	samba? ( net-fs/samba )"
+
+DOCS=( "CREDITS.md" "TODO" )
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-parallel-make.patch"
+	"${FILESDIR}/${P}-use-freetype2-pkg-config.patch"
+)
+
+src_prepare() {
+	default
+
+	# Fix path for modules
+	sed -e "37s:server/drivers:/usr/$(get_libdir)/lcdproc:" -i LCDd.conf || die
+
+	# Copy missing docbook or the buildung of the lcdproc-user docbook will fail
+	cp "${FILESDIR}"/linux_input.docbook "${S}"/docs/lcdproc-user/drivers/linux_input.docbook || die
```

Comment:
Please move it to SRC_URI, so we wouldn't have to carry such huge, copyright-unclear files in the repo.

Reply:
Thanks! Sounds much better, done.